src/libAtomVM/defaultatoms.c

```diff
@@ -161,6 +161,11 @@ static const char *const cast_atom = "\x5" "$cast";
 static const char *const unicode_atom = "\x7" "unicode";
 
 static const char *const global_atom = "\x6" "global";
+static const char *const type_atom = "\x4" "type";
+static const char *const name_atom = "\x4" "name";
+static const char *const arity_atom = "\x5" "arity";
+static const char *const external_atom = "\x8" "external";
+static const char *const local_atom = "\x5" "local";
```

Comment:
Formatter chokes on this file and few others. Is it okay to submit a PR formatting some of them or you prefer not introducing unnecessary blames? For the record, there's also exist `.git-blame-ignore-revs` file but requires a bit of git config.

Reply:
There is an `ATOM_STR` that avoids that issue. Let's stay consistent with the rest of the file now, we'll make use of the ATOM_STR macro also here sooner or later.

Reply:
If you want, I can submit a PR to do that. It will probably introduce conflicts for other PRs so maybe it's better to wait until we have less PRs unmerged.

Reply:
yes, agreed.